src/pages/GE_TokenizeCard.page

```diff
@@ -0,0 +1,94 @@
+<apex:page showHeader="false" applyHtmlTag="true" applyBodyTag="false">
+
+    <head>
+        <apex:slds />
+    </head>
+
+    <article class='slds-scope'>
+        <form onsubmit="handleForm(event)">
+            <div id="payments"></div>
+        </form>
+    </article>
+
+    <!-- TODO: Update src with production url and/or upload as static resource -->
+    <script src="https://d2w37eyz6rciar.cloudfront.net/elevate.min.js"></script>
+
+    <script>
+        const hostname = window.location.hostname;
+        const arr = hostname.split(".");
+        let domain = arr[0].replace('--npsp', '');
+        domain = domain.replace('--c', '');
+        const lightningOrigin = `https://${domain}.lightning.force.com`;
+
+        const sfdo = new sfdoPaymentsJsSdk();
+        sfdo.mount({
+            id: 'payments',
+            clientId: '00D6g0000000000EAA',
```

Comment:
Can we add a reminder comment here to use {!$Organization.Id} when that is going to be ready?